bitnami/external-dns/values-production.yaml

```diff
@@ -111,6 +111,9 @@ aws:
   ## AWS Role to assume
   ##
   assumeRoleArn: ""
+  ## Maximum number of retries for AWS API calls before giving up.
+  ##
+  apiRetries: 10
```

Comment:
We're using `external-dns` on a large platform that got throttle a lot from AWS API Route53 call limit (mainly due to non-optimum design, but that's another story):

```
time="2020-11-03T12:57:43Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 35312ab6-9965-42d1-8795-ae4e05b73a93"
time="2020-11-03T12:58:27Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: aaaa80cf-7985-4865-b2e5-1f143ecc7963"
time="2020-11-03T13:00:32Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 37f41c31-c41c-4cb6-b3d8-c68be37c6fab"
time="2020-11-03T13:00:38Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 3ad24117-5fb0-4755-b4ba-44edf0ec012a"
time="2020-11-03T13:03:04Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 83f5e078-2b31-477c-b840-321f3bb3b036"
time="2020-11-03T13:03:15Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 65c01971-0d1c-463e-be70-108aef52a156"
time="2020-11-03T13:04:10Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 062e69da-f0d1-45dc-8621-ed0a5d0fb146"
time="2020-11-03T13:05:11Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 7e59d187-c046-4ce5-92ba-7afbca29710e"
time="2020-11-03T13:06:14Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: eb4ef995-17e4-45a6-9cd9-58be425aa6ae"
time="2020-11-03T13:07:33Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 668770f3-fb96-4d0a-b1ee-d0d6711c0009"
time="2020-11-03T13:09:30Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 02e3eaa6-1222-45b9-ab24-92a14f3a1e95"
time="2020-11-03T13:10:46Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 1869a103-5c18-4ca3-9cfa-1e12e483df27"
time="2020-11-03T13:10:51Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: d5e262e7-d347-47ff-9469-63c6b789aeaa"
time="2020-11-03T13:12:18Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 3c5faa30-899a-4713-be67-b46711663706"
time="2020-11-03T13:13:00Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: c05de41f-76c4-4330-b825-775447c59792"
time="2020-11-03T13:14:00Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: a00d79de-bfe3-479a-b797-3db83ece9996"
time="2020-11-03T13:15:17Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 14f5b1ec-fb02-439e-a223-b09414f99df6"
time="2020-11-03T13:16:09Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 18568702-00ae-4e5a-bc6e-41c4d78b1378"
time="2020-11-03T13:18:20Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 29a22605-e945-412c-a5d4-b4bbb3babfe5"
time="2020-11-03T13:18:27Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 0b17974c-a09a-4e88-ba11-88c0402aecdc"
time="2020-11-03T13:20:45Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 1674213e-9dd1-4724-a9df-c51d84dc1921"
time="2020-11-03T13:21:25Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 591ff6fa-f336-463d-84c9-36fd82ad82e5"
time="2020-11-03T13:22:10Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 9e887b5a-065b-493c-8626-23188cad605c"
time="2020-11-03T13:23:13Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 371e62a0-9fc6-4c24-9de8-2d249f16c036"
time="2020-11-03T13:24:05Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 9fa4daf5-1348-47e5-9509-33b2984128b7"
time="2020-11-03T13:25:03Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 7fb14b53-a66e-44a4-9860-65b3dc5aa28b"
time="2020-11-03T13:25:56Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: be1b0bda-fbce-410d-af5d-51a4f8101bba"
time="2020-11-03T13:27:38Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: e1cdebe8-2536-4dbe-928b-86c67f90d2f4"
time="2020-11-03T13:27:57Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: e3408f81-98e4-4f6b-a835-2502ccb72637"
time="2020-11-03T13:32:36Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 33ff9604-3ecf-42df-b3ed-2deb733abdfb"
time="2020-11-03T13:32:43Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: df1c7f46-fc10-49dd-a852-bd0a2d7133b4"
time="2020-11-03T13:34:17Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: fb37159b-aaf4-4fbe-bb36-3b40c744a08c"
time="2020-11-03T13:34:45Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 529de0de-e800-4f0a-adf3-a0669b2d5532"
time="2020-11-03T13:36:04Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: 9bb6d6ae-75b7-45c5-86f5-c66a868da298"
time="2020-11-03T13:36:57Z" level=error msg="Throttling: Rate exceeded\n\tstatus code: 400, request id: cfcfc50b-d466-4280-9376-c39ead56e5ee"
```

Bumping to 10 allow `external-dns` to fulfil its work (we tried 5-7 without success before :( )

Reply:
If it's preferable, I can switch back to default value 3 in the default `values*.yaml`, to avoid any mismatch (we'll keep using 10 on our side surcharging it from `values.yaml`)